src/components/dashboard/dashboard.service.ts

```diff
@@ -436,6 +440,22 @@ export class DashboardService implements OnDestroy {
             dimensions.height = this.options.minHeight;
         }
 
+        const minWidth = this.getColumnWidth() * action.widget.minColSpan - this.getColumnWidth() / 2 + 1;
```

Comment:
Could we add some parenthesis here and for the `minHeight` calculation too. We are heavily relying on operator precedence and at a glance it is not obvious what order this is running in, e.g.

```suggestion
        const minWidth = ((this.getColumnWidth() * action.widget.minColSpan) - (this.getColumnWidth() / 2)) + 1;
```